sgxs-loaders/src/isgx/mod.rs

```diff
@@ -132,13 +207,74 @@ impl EnclaveLoad for InnerDevice {
             flags: eadd.secinfo.flags,
             ..Default::default()
         };
-        let adddata = ioctl::AddData {
-            dstpage: mapping.base + eadd.offset,
-            srcpage: &data,
-            secinfo: &secinfo,
-            chunks: chunks.0,
-        };
-        ioctl_unsafe!(Add, ioctl::add(mapping.device.fd.as_raw_fd(), &adddata))
+        let dstpage = mapping.base + eadd.offset;
+        match mapping.device.driver {
+            Montgomery => {
+                let adddata = ioctl::montgomery::AddData {
+                    dstpage,
+                    srcpage: &data,
+                    secinfo: &secinfo,
+                    chunks: chunks.0,
+                };
+                ioctl_unsafe!(Add, ioctl::montgomery::add(mapping.device.fd.as_raw_fd(), &adddata))
+            },
+            Augusta => {
+                let flags = match chunks.0 {
+                    0 => ioctl::augusta::SgxPageFlags::empty(),
+                    0xffff => ioctl::augusta::SgxPageFlags::SGX_PAGE_MEASURE,
```

Comment:
Where do these constants come from?

Reply:
It's just “no bits set” and “all bits set” meaning all or no 256-byte chunks in a page, see also the Windows loader.